<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-5</y>
      <width>1061</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="choosePartWidget">
     <widget class="QTabWidget" name="choosePartTabs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>1061</width>
        <height>641</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="choosePartTab">
       <attribute name="title">
        <string>Choose Part</string>
       </attribute>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>60</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Price(KRW)</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ramCombo">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>280</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>460</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Power</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mainboard</string>
        </property>
       </widget>
       <widget class="QLabel" name="ramPrice">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>280</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>520</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sum :</string>
        </property>
       </widget>
       <widget class="QLabel" name="hddPrice">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>400</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="vgaPrice">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>220</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="sumPrice">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>520</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ssdPrice">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>340</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SSD</string>
        </property>
       </widget>
       <widget class="QPushButton" name="resultCombination">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>550</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Performance</string>
        </property>
       </widget>
       <widget class="QComboBox" name="vgaCombo">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cpuCombo">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
       <widget class="QComboBox" name="hddCombo">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>400</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>VGA</string>
        </property>
       </widget>
       <widget class="QComboBox" name="mbCombo">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RAM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>HDD</string>
        </property>
       </widget>
       <widget class="QComboBox" name="powerCombo">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>460</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="ssdCombo">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>340</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="cpuPrice">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>100</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="powerPrice">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>460</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="mbPrice">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>160</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="initCombination">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>550</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="cpuDescription">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>100</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
         </font>
        </property>
       </widget>
       <widget class="QTextBrowser" name="mbDescription">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>160</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
         </font>
        </property>
       </widget>
       <widget class="QTextBrowser" name="vgaDescription">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>220</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
         </font>
        </property>
       </widget>
       <widget class="QTextBrowser" name="ramDescription">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>280</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
         </font>
        </property>
       </widget>
       <widget class="QTextBrowser" name="ssdDescription">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>340</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
         </font>
        </property>
       </widget>
       <widget class="QTextBrowser" name="hddDescription">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>400</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
         </font>
        </property>
       </widget>
       <widget class="QTextBrowser" name="powerDescription">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>460</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="goToGameSelection">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>10</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Game Selection</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Part Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>60</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="chooseGameWidget">
     <widget class="QTabWidget" name="chooseGameTabs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>1061</width>
        <height>641</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="chooseGame">
       <attribute name="title">
        <string>Game Selection</string>
       </attribute>
       <widget class="QListWidget" name="gameList">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>411</width>
          <height>501</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="selectedList">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>70</y>
          <width>411</width>
          <height>501</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>40</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Game List</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>40</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Selection List</string>
        </property>
       </widget>
       <widget class="QPushButton" name="gameAdd">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>80</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="gameDelete">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>130</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="gameClear">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
       <widget class="QPushButton" name="recommend">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>230</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Recommend</string>
        </property>
       </widget>
       <widget class="QPushButton" name="goToPartSelection">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>10</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Part Selection</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="quoteTab">
       <attribute name="title">
        <string>Recommendation</string>
       </attribute>
       <widget class="QTableWidget" name="partRecTable">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>11</y>
          <width>1011</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="retry">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>540</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Retry</string>
        </property>
       </widget>
       <widget class="QPushButton" name="reselect">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>540</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Game Selection</string>
        </property>
       </widget>
       <widget class="QPushButton" name="printRecParts">
        <property name="geometry">
         <rect>
          <x>930</x>
          <y>540</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
